lib/components/viewers/stop-schedule-viewer.tsx

```diff
@@ -84,6 +81,46 @@ const StyledAlert = styled(Alert)`
   text-align: center;
 `
 
+const HeaderCard = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 5px 0 0;
+
+  ${CardBody} {
+    margin: 25px 0 0;
+  }
+
+  input[type='date'] {
+    background: inherit;
+    border: none;
+    clear: right;
+    cursor: pointer;
+    outline: none;
+    width: 125px;
```

Comment:
Is there a reason this width is fixed? It looks like it's causing a little gaping between the icon and the input on Chrome.
<img width="197" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/115499534/975ba646-138f-4c4f-8af6-5b6adee1d289">

Reply:
In Firefox, the text field takes up more space.